multimodal/src/autogluon/multimodal/data/process_image.py

```diff
@@ -102,8 +114,22 @@ def __init__(
         self.mean = None
         self.std = None
 
+        if self.prefix == MMDET_IMAGE:
+            # TODO: can we pass the config information here when we build the model?
+            try:
+                cfg = checkpoint_name + ".py"
+                if isinstance(cfg, str):
+                    cfg = mmcv.Config.fromfile(cfg)
+            except:
+                raise RuntimeError(
+                    "If encounterd mmcv related error, please install mmcv-full by: mim install mmcv-full."
+                )
```

Comment:
Let's check if mmcv and mmdet is None here.
Currently, with this try except, it's possible that mmcv/mmdet is not installed is not the problem. For example, what if the cfg file is invalid.

Maybe we can do an assert direct, i.e. `assert mmcv is not None and mmdet is not None, "ERROR MSG"`

Reply:
Yes, good point. Already added assert